python/pyspark/pandas/frame.py

```diff
@@ -8164,6 +8164,89 @@ def update(self, other: "DataFrame", join: str = "left", overwrite: bool = True)
         internal = self._internal.with_new_sdf(sdf, data_fields=data_fields)
         self._update_internal_frame(internal, requires_same_anchor=False)
 
+    def cov(self, min_periods: Optional[int] = None) -> "DataFrame":
+        """
+        Compute pairwise covariance of columns, excluding NA/null values.
+        Compute the pairwise covariance among the series of a DataFrame.
+        The returned data frame is the `covariance matrix
+        <https://en.wikipedia.org/wiki/Covariance_matrix>`__ of the columns
+        of the DataFrame.
+        Both NA and null values are automatically excluded from the
+        calculation. (See the note below about bias from missing values.)
+        A threshold can be set for the minimum number of
+        observations for each value created. Comparisons with observations
+        below this threshold will be returned as ``NaN``.
+        This method is generally used for the analysis of time series data to
+        understand the relationship between different measures
+        across time.
```

Comment:
nit: Can we add a newline between the sentence such as pandas does ??

<img width="797" alt="Screen Shot 2021-09-07 at 4 07 55 PM" src="https://user-images.githubusercontent.com/44108233/132299963-b99a6514-06f2-488d-8a89-b9146c4c2b8c.png">

Reply:
thanks, updated